docs/examples/calc_jos3.html

```diff
@@ -0,0 +1,203 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>JOS3 example</title>
+    <meta
+      name="description"
+      content="Example of JOS3 Model usage in jsthermalcomfort"
+    />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <script
+      src="https://cdn.plot.ly/plotly-2.26.1.min.js"
+      charset="utf-8"
+    ></script>
+  </head>
+  <body style="text-align: center">
+    <h1>JOS3 example</h1>
+    <div>
+      <h2>EXAMPLE 1 (simple simulation)</h2>
+      <div id="graph1" style="width: 600px; height: 600px; margin: auto"></div>
+      <h style="align-items: center"
+        >The BMR value using the getter:
+        <div id="example1 bmr"></div
+      ></h>
+      <h style="align-items: center"
+        >The body name lists:
+        <div id="example1 Body name list"></div
+      ></h>
+    </div>
+
+    <div>
+      <h2>EXAMPLE 2 (detail simulation)</h2>
+      <div id="graph2" style="width: 600px; height: 600px; margin: auto"></div>
+    </div>
+  </body>
+  <script type="module">
+    import { JOS3 } from "https://cdn.jsdelivr.net/gh/FedericoTartarini/jsthermalcomfort/lib/esm/models/JOS3.js";
+    //from pythermalcomfort.jos3_functions.utilities import local_clo_typical_ensembles
+    import { local_clo_typical_ensembles } from "https://cdn.jsdelivr.net/gh/FedericoTartarini/jsthermalcomfort/lib/esm/jos3_functions/utilities.js";
+
+    let model = new JOS3(1.7, 60, undefined, 30);
+    // Set the first phase
+    model.to = 28; // Operative temperature [°C]
+    model.rh = 40; // Relative humidity [%]
+    model.v = 0.2; // Air velocity [m/s]
+    model.par = 1.2; // Physical activity ratio [-]
+    model.simulate(60); // Exposure time = 60 [min]
+
+    // Set the next condition (You only need to change the parameters that you want to change)
+    model.to = 20; // Change only operative temperature
+    model.simulate(60); // Additional exposure time = 60 [min]
+
+    let t_skin_mean_trace = {
+      x: model.cycle_time,
+      y: model.t_skin_mean,
+      mode: "lines",
+    }; // Plot time series of mean skin temperature.
+    let layout = {
+      xaxis: {
+        title: "Time [min]",
+      },
+      yaxis: {
+        title: "Mean skin temperature [°C]",
+      },
+    };
+    Plotly.newPlot("graph1", [t_skin_mean_trace], layout);
+
+    // Print the BMR value using the getter
+    document.getElementById("example1 bmr").innerHTML = model.bmr;
+    document.getElementById("example1 Body name list").innerHTML =
+      model.body_names;
```

Comment:
```suggestion
    model.body_names;
```
I want to test something